docs/_specification/1.2-DRAFT/data-entities.md

```diff
@@ -416,9 +507,62 @@ Alternatively, a common mechanism to provide downloads of a reasonably sized dir
   }
 ```
 
-Similarly, the _RO-Crate root_ entity may also provide a [distribution] URL, in which case the download SHOULD be an archive that contains the _RO-Crate Metadata Document_.
+Similarly, the _RO-Crate root_ entity (or a reference to another RO-Crate as a `Dataset`) may provide a [distribution] URL, in which case the download SHOULD be an archive that contains the _RO-Crate Metadata Document_ (either directly in the archive's root, or within a single folder in the archive), indicated by a version-less `conformsTo`:
+
+```json
+  {
+    "@id": "./",
+    "@type": "Dataset",
+    "identifier": "https://doi.org/10.48546/workflowhub.workflow.775.1",
+    "name": "Research Object Crate for Jupyter Notebook Molecular Structure Checking",
+    "distribution": {"@id": "https://workflowhub.eu/workflows/775/ro_crate?version=1"},
+    "…": ""
+  },
+  {
+    "@id": "https://workflowhub.eu/workflows/775/ro_crate?version=1",
+    "@type": "DataDownload",
+    "encodingFormat": ["application/zip", {"@id": "https://www.nationalarchives.gov.uk/PRONOM/x-fmt/263"}],
+    "conformsTo": { "@id": "https://w3id.org/ro/crate" }
```

Comment:
I understand that this is needed to be correct describing downloadable RO-Crate. But, in this scenario, if the graph does not contain an `ro-crate-metadata.json` (or similar) node, or contains it without a proper `conformsTo`, shall we assume that metadata should appear at the RO-Crate fetchable from the downloadable content?

Reply:
This is a cheeky triple-use of our `conformsTo` as we define what the profile mean on different files.. so here the implication is that if there is `https://w3id.org/ro/crate` as profile of a `DataDownload`  then yes, there should be a `ro-crate-metadata.json` inside it's archive, which presumably would have its own (versioned) `conformsTo` declared.   Or if it's a `Dataset` with the same `conformsTo` then the metadata file is found differently.

We also use the same profile as part of the JSON-LD content type https://www.researchobject.org/ro-crate/specification/1.2-DRAFT/appendix/jsonld.html#ro-crate-json-ld-media-type 

You could argue this should be several different profile URIs.. I think it's a bit overloading, but if you squeeze your eyes a bit it's all saying "an RO-Crate" just in different ways in.

But we won't try to list the content of the zip-file in the referring crate, unless we need to (which would bring in `arcp` etc), so that's a nice simplification that we moved to having the references on the `Dataset` rather than requiring mention of the second RO-Crate Metadata Document.

Reply:
https://signposting.org/adopters/#workflowhub Signposting also uses the same profile for the ZIP file. But that should also be valid to mean "There will be an `ro-crate-metadata.json` inside, as stated in top of section "#### Retrieving an RO-Crate"